src/js/components/returns/inbound/AddItemsPage.jsx

```diff
@@ -273,33 +322,13 @@ class AddItemsPage extends Component {
     }
   }
 
-  nextPage(formValues) {
-    const returnItems = _.filter(formValues.returnItems, val => !_.isEmpty(val) && val.product);
-
-    if (_.some(returnItems, item => !item.quantity || item.quantity === '0')) {
-      this.confirmSave(() =>
-        this.saveStockTransfer(returnItems, this.state.inboundReturn.status !== 'PLACED' ? 'PLACED' : null)
-          .then(() => {
-            this.props.nextPage(this.state.inboundReturn);
-          })
-          .catch(() => this.props.hideSpinner()));
-    } else {
-      this.saveStockTransfer(returnItems, this.state.inboundReturn.status !== 'PLACED' ? 'PLACED' : null)
-        .then(() => {
-          this.props.nextPage(this.state.inboundReturn);
-        })
-        .catch(() => this.props.hideSpinner());
-    }
-  }
-
-  saveStockTransferInCurrentStep(returnItems) {
-    this.props.showSpinner();
-    return this.saveStockTransfer(returnItems, null)
-      .catch(() => this.props.hideSpinner());
+  async nextPage(formValues) {
+    this.saveStockTransferInCurrentStep(formValues, this.state.inboundReturn.status !== 'PLACED' ? 'PLACED' : null)
```

Comment:
I think it will be worth creating an enum with statuses including this `PLACED` instead of using magic strings